api/endpoints/community.py

```diff
@@ -36,7 +38,9 @@ def make_blueprint(con):
         "rent_burdened_2019": lambda: metric_rbu.rent_burdened(year=2019, segment="all"),
         "rent_max": lambda: metric_rbu.rent_max_burdened(),
         "rent_min": lambda: metric_rbu.rent_min_burdened(),
-        "rent_average": lambda: metric_rbu.rent_average_burden_area()
+        "rent_average": lambda: metric_rbu.rent_average_burden_area(),
+        "avg_speed_per_dropoff": lambda: metric_tt.get_avg_speed_per_dropoff(),
+        "avg_speed_per_pickup": lambda: metric_tt.get_avg_speed_per_pickup()
```

Comment:
This works, but technically this doesn't need to be a lambda function.

The reason we use lambda functions in the supported_metrics map is that sometimes, we want to modify a query with a variable, but we don't want to write an entirely new method. However, supported_metrics expects a method with no parameters. So we use a lambda function to create a method that has no parameters by "wrapping" our call to the method with the parameter we want. This is called "currying."

However, your avg_speed methods do not take any parameters, so you could list them like this, with no parentheses after the method, which passes the method as a variable, rather than calling it:

```python
{
        ...
        "avg_speed_per_dropoff": metric_tt.get_avg_speed_per_dropoff,
        "avg_speed_per_pickup": metric_tt.get_avg_speed_per_pickup
}
```

No need to change it though, I noticed just now that the same thing was done for the rent burdened metrics.